workers/tests/integrationtests/fixtures/vcr_cassettes/indicators/test_building_completeness.yaml

```diff
@@ -70,4 +70,75 @@ interactions:
       - accept-encoding
```

Comment:
I am wondering why this request is add as vcr cassette? Should not everything stay the same from `indicator.py` perspective?